### PR TITLE
Add DBusPolicyCheck from openSUSE/rpmlint-checks (#227).

### PR DESCRIPTION
**Pull request by marxin**

### CODE REVIEW

File: rpmlint/checks/DBusPolicyCheck.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            if f in pkg.ghostFiles():
+                continue
+
+            # catch xml exceptions
+            try:
+                if f.startswith('/etc/dbus-1/system.d/'):
+                    send_policy_seen = False
+                    lf = pkg.dirName() + f
+                    xml = parse(lf)
+                    for p in xml.getElementsByTagName('policy'):
```
Comment (scarabeusiv) [inline rpmlint/checks/DBusPolicyCheck.py]: This is getting way too nested, could it be separated to functions please?

Comment (marxin) [reply rpmlint/checks/DBusPolicyCheck.py]: Yep. Fixed.

**Comment by scarabeusiv** [comment]
This one I would recommend for default enabled treatment.

File: rpmlint/checks/DBusPolicyCheck.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                send_policy_seen = True
+
+            if (allow.hasAttribute('receive_sender') or
+                    allow.hasAttribute('receive_interface')):
+                self.output.add_info('W', pkg, 'dbus-policy-allow-receive', allow.toxml(), f)
+
+        for deny in policy.getElementsByTagName('deny'):
+            if (deny.hasAttribute('send_interface') and not
+                    deny.hasAttribute('send_destination')):
+                self.output.add_info('E', pkg, 'dbus-policy-deny-without-destination', allow.toxml(), f)
```
Comment (scarabeusiv) [inline rpmlint/checks/DBusPolicyCheck.py]: This for loop could be just one separate function as it is not tied to the policy_seen at all.

Comment (marxin) [reply rpmlint/checks/DBusPolicyCheck.py]: Done, actually I found an issue in `dbus-policy-deny-without-destination` error message where I used `allow.toxml()`.

**Pull request merged**